### PR TITLE
Feature eod reminder parser

### PR DESCRIPTION
**Pull request by naiuhz**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 node_modules
 npm-debug.log
-dist
\ No newline at end of file
+dist
+
+!/config-files/.gitkeep
```
Comment (manekenpix) [inline .dockerignore]: Add next line

File: .gitignore [hunk 2]
```diff
@@ -6,4 +6,6 @@
 /eods.json
 *secret*
 /src/clock.txt
-*.env
\ No newline at end of file
+*.env
+/config-files/*
+!/config-files/.gitkeep
```
Comment (manekenpix) [inline .gitignore]: Add next line

File: src/osteppy_server_setup.sh [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+cp ../config-files/osteppy.service /usr/lib/systemd/system/
+
+# Reload
+systemctl daemon-reload
+
+# Enable Services
+systemctl enable osteppy
+
+# Start Services
+systemctl start osteppy
```
Comment (manekenpix) [inline src/osteppy_server_setup.sh]: Add next line

File: .gitignore (hunk 2 above)
Comment (naiuhz) [reply .gitignore]: Add next line to the end?

File: src/EODReminder.js [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+const fs = require('fs');
+const path = require('path');
+
+const { execSync } = require('child_process');
+const { WebClient } = require('@slack/client');
+const cpCommand = 'cp ../config-files/RAs.txt ../config-files/sleepyRAs.txt';
+const slackToken = fs.readFileSync(path.resolve(__dirname, '../config-files/SLACK_TOKEN'), 'utf8');
+const web = new WebClient(slackToken);
+const channelIDs = require('../config-files/channelID.json');
+const EODReminders = require('../config-files/EODReminderTimes.json');
```
Comment (Tsu-ba-me) [inline src/EODReminder.js]: camelCase, i.e., eodReminders
(hint: CTRL+F2 in VSCode)

File: src/EODReminder.js [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+        if (reminder["weekday"].includes(today.getDay()) && reminderTime[0] === nextReminderTime[0] && reminderTime[1] === nextReminderTime[1]){
+          nextReminders.push([name, reminder["message"]]);
+        }
+      }
+    }
+  });
+
+  //console.log ("DEBUG nextReminders: " + nextReminders);
+
+  setTimeout(function(){sendEODs(nextReminders);}, nextReminderSeconds);
```
Comment (Tsu-ba-me) [inline src/EODReminder.js]: Use lambda syntax?
`setTimeout(() => sendEODs(nextReminders), nextReminderSeconds)`

File: src/EODReminder.js [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+  let nextReminderTime = Array(2);
+  let nextReminders = [];
+  sleepyRANames.forEach((name) => {
+    if (EODReminders[name] !== undefined){
+      let reminderArr = EODReminders[name]["reminders"];
+      for (var i = 0; i < reminderArr.length; i++){
+        let reminder = reminderArr[i];
+        if (reminder["weekday"].includes(today.getDay())){
+          let reminderTime = reminder["time"].split(":");
+          let reminderDate = new Date(today.getFullYear(), today.getMonth(), today.getDate(), reminderTime[0], reminderTime[1], 0);
```
Comment (poftadeh) [inline src/EODReminder.js]: Rename any `let` to `const` if its value is not being changed.

**Comment by naiuhz** [comment]
I resolved those suggestions, please accept the PR.

**Pull request merged**